lib/LedgerSMB/Scripts/asset.pm

```diff
@@ -1010,7 +1010,18 @@ sub run_import {
 }
 
 ###TODO-LOCALIZE-DOLLAR-AT
-eval { do "scripts/custom/asset.pl"};
+eval {
```

Comment:
these eval{}s are here to suppress errors such as files not existing. Probably *just* to suppress the file-does-not-exist error.

Since we prevent that error now, I'd say that we don't need to wrap an eval{} around the code block.